Client/Extensions/UIAlertControllerExtensions.swift

```diff
@@ -187,4 +195,106 @@ extension UIAlertController {
 
         return deleteAlert
     }
-}
+
+    /**
+     Builds the Alert view that asks if the users wants to add a third party search engine.
+
+     - parameter okayCallback: Okay option handler.
+
+     - returns: UIAlertController for asking the user to add a search engine
+     **/
+
+    class func addThirdPartySearchEngine(okayCallback: (UIAlertAction) -> Void) -> UIAlertController {
+        let alert = UIAlertController(
+            title: NSLocalizedString("AddTitle", tableName: "addSearchEngine", comment: "The title that asks the user to Add the search provider"),
```

Comment:
Looks like you're using a placeholder. What's the actual string that we want here?